atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -374,6 +329,42 @@ def update_repos_from_composes(self):
             self.log.info('adding repo file from compose: %s', compose_info['result_repofile'])
             self.repos.append(compose_info['result_repofile'])
 
+    def _add_filesystem_to_dockerfile(self, file_name, build_dir: BuildDir):
+        """
+        Put an ADD instruction into the Dockerfile (to include the filesystem
+        into the container image to be built)
+        """
+        content = 'ADD {0} /\n'.format(file_name)
+        lines = build_dir.dockerfile.lines
+        # as we insert elements we have to keep track of the increment for inserting
+        increment = 1
+
+        for item in build_dir.dockerfile.structure:
+            if item['instruction'] == 'FROM' and base_image_is_custom(item['value'].split(' ')[0]):
```

Comment:
```suggestion
            if item['instruction'] == 'FROM' and base_image_is_custom(item['value'].split()[0]):
```
Would suggest to split on any whitespace, this will probably never be relevant but just to be sure

Reply:
used suggestion